spec/migrations/20180405145642_remove_provider_region_for_google_provider_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_migration
+
+describe RemoveProviderRegionForGoogleProvider do
+  let(:ems_stub) { migration_stub :ExtManagementSystem }
+
+  migration_context :up do
+    before do
+      ems_stub.create!(:type            => 'ManageIQ::Providers::Google::CloudManager',
+                       :provider_region => 42)
+      ems_stub.create!(:type            => 'ManageIQ::Providers::Amazon::CloudManager',
+                       :provider_region => 42)
+      ems_stub.create!(:type            => 'ManageIQ::Providers::Azure::CloudManager',
+                       :provider_region => 42)
+      ems_stub.create!(:type            => 'ManageIQ::Providers::Openstack::CloudManager',
+                       :provider_region => 42)
+      ems_stub.create!(:type            => 'ManageIQ::Providers::Vmware::CloudManager',
+                       :provider_region => 42)
+
+      migrate
```

Comment:
@ZitaNemeckova 

- I would join your `its` to one it with `expect(` because it would be nice to keep one `it` case for one migrating process and adapt 'it' message.

- I would also prefer creating object in lets although it is not needed in this case if there will be only one it.
- move `migrate` command to the `it` and remove before block.

What do you think about this suggestions ?